sigstore-java/src/main/java/dev/sigstore/bundle/BundleWriter.java

```diff
@@ -53,6 +53,8 @@ static String writeBundle(Bundle signingResult) {
     try {
       String jsonBundle = JSON_PRINTER.print(bundle);
       List<String> missingFields = BundleVerifier.findMissingFields(bundle);
+      // integrated_time is a required field in the Bundle spec
```

Comment:
can we add a TODO(#bug-number) on this to update once that field becomes optional?